### DISCUSSION TITLE
Image Editor

### DISCUSSION THREAD

**Discussion by Frozenpath** [awcodes/filament-tiptap-editor#184] [Q&A]
Firstly I want to say thanks for this really cool package.

My question about ability to add Image Editor like in standart Filament File Uploader field. I think it's really usefull feature and it will be nice to have it here too.

https://filamentphp.com/docs/3.x/forms/fields/file-upload#image-editor

![image](https://github.com/awcodes/filament-tiptap-editor/assets/26181174/2b482cbc-04ac-4d7b-a4ed-ad4242d3c072)

**Comment by awcodes** [comment]
hmm. I"m open to PRs. :)

I'm not even totally 100% sure it will even work. But I'll make a note to explore it when I get some time.
